lib/ws/ws.js

```diff
@@ -485,7 +492,8 @@ BeamSocket.prototype.close = function () {
  * we try to require Bluebird, but you can switch it out with your own.
  * @type {Promise}
  */
-try { BeamSocket.Promise = require('bluebird'); } catch (e) {}
+// eslint-disable-next-line global-require
```

Comment:
The rule in general is a good indicator that you are doing something wrong (aside from tests).